setup.py

```diff
@@ -40,6 +40,7 @@ def load_benchmark_requirements():
 required_aws = load_requirements("requirements-aws.txt")
 required_moo = load_requirements("requirements-moo.txt")
 required_visual = load_requirements("requirements-visual.txt")
+required_sklearn = load_requirements("requirements-sklearn.txt")
```

Comment:
This will collect requirements of the scikit-learn based surrogate models

Reply:
Thats a nice change, thanks